js/stores/appStore.js

```diff
@@ -253,7 +257,7 @@ function windowDefaults () {
     show: false,
     width: appState.get('defaultWindowWidth'),
     height: appState.get('defaultWindowHeight'),
-    minWidth: 480,
+    minWidth: isWindows ? 640 : 480, // Windows has caption buttons (min/max/close) which take extra space
```

Comment:
OK will check those out- I haven't had success running the webdriver tests on Windows (yet)